.github/workflows/CodeCoveraReport.yaml

```diff
@@ -0,0 +1,77 @@
+name: Code coverage report to Codecov
+
+on:
+    push:
+      branches: 
+        - master
+jobs:
+        
+  test_client:
+    runs-on: ubuntu-latest
+        
+    name: Create and upload coverage
+    steps:
+        
+      - name: Setup Java
+        uses: actions/setup-java@v1
+        with:
+          java-version: 8
+            
+      - name: Checkout code           
+        uses: actions/checkout@v2
+        
+      - name: Install Necessary Packages
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y net-tools libssl-dev gdb
+          
+      - name: Install Boost
+        run: |
+          sudo ./scripts/install-boost.sh 1.76.0
+          
+      - name: Install Thrift
+        run: |
+          sudo ./scripts/install-thrift.sh 0.13.0
+         
+      - name: Setup Gcovr   
+        run: |
+            sudo apt install gcovr
```

Comment:
Can we add `gcovr` to the "Install Necessary Packages" step above and remove this step?

Reply:
make sense, fixing